FluentTc/Engine/BuildConfigurationRunner.cs

```diff
@@ -145,6 +145,15 @@ private static string CreateTriggerBody(string buildConfigId, int? agentId, List
                 bodyBuilder.Append(@"</properties>").AppendLine();
             }
 
+            if (moreOptions != null && moreOptions.GetChangeId() != 0)
```

Comment:
Maybe it would be better to use an explicit method to check whether moreOptions has ChangeId for example 
```C#
moreOptions.HasChangeId()
```
instead of 
```C#
moreOptions.GetChangeId() != 0
```